libs/types/src/lib/types.ts

```diff
@@ -42,9 +42,10 @@ export interface IEvent {
 }
 
 export interface ILocation {
-  id: number;
-  point: string;
-  events?: IEvent[];
+  id?: number;
+  title: string;
+  placeId?: string;
```

Comment:
Wait what is place id?

Reply:
To explain it briefly, the google places api (autocomplete) which we're using doesn't actually give back coordinates, instead it gives back a placeId that we can use to get back the location with geocoding 

https://developers.google.com/maps/documentation/javascript/geocoding

Reply:
![image](https://user-images.githubusercontent.com/35787279/102731104-f4261a00-4304-11eb-8454-7d36fe68d269.png)

Reply:
So currently the user flow now goes as follows:

User presses Current location --> we'll use geolocation api to get back their [Lat,Long]

User types in an address and selects one --> We get back the string and its associated placeId

Reply:
Looked into this with @JimLau49, we can use https://developers.google.com/places/web-service/search to search for places too, which will give us the lat/lng right off the bat. The search is smart (you don't need to type out the entire address to get the result) but it doesn't autocomplete (you need to type out the street name for example).

Unfortunately, for places autocomplete, there is no option to get the lat/lng, so we have to compromise on something.